tooling/cli/src/command/tokens/create-theme-typings-interface.ts

```diff
@@ -37,11 +37,41 @@ export interface ThemeKeyOptions {
   flatMap?: (value: string) => string | string[]
 }
 
+export type TypingsTemplate = "default" | "augmentation"
+
 export interface CreateThemeTypingsInterfaceOptions {
   config: ThemeKeyOptions[]
   strictComponentTypes?: boolean
   format?: boolean
   strictTokenTypes?: boolean
+  template?: TypingsTemplate
+}
+
+function applyThemeTypingTemplate(
+  typingContent: string,
+  template: TypingsTemplate,
+) {
+  switch (template) {
+    case "augmentation":
+      return `// regenerate by running
+// npx @chakra-ui/cli tokens path/to/your/theme.(js|ts)
```

Comment:
**suggestion**: Since we have two different template we could provide the correct command to generate it as well.

```suggestion
// npx @chakra-ui/cli tokens --template augmentation --out path/to/this/file path/to/your/theme.(js|ts)
```